gui/ThemeEval.cpp

```diff
@@ -50,7 +50,7 @@ void ThemeEval::reset() {
 	_layouts.clear();
 }
 
-bool ThemeEval::getWidgetData(const Common::String &widget, int16 &x, int16 &y, int16 &w, int16 &h) {
+bool ThemeEval::getWidgetData(const Common::String &widget, int16 &x, int16 &y, int16 &w, int16 &h, bool &useRTL) {
```

Comment:
@sev- Should useRTL have a default value here since this method is used in Engines to create dialogs.

Reply:
IMO we should use default Graphics::TextAlignment in widgets, and only use the useRTL flag in GUI code to tell the GUI to set the default to kTextAlignRight

Reply:
Guys, this is a _getter_, e.g. it tells what is specified in the XML theme. Of course, there is a default present, in the constructor of ThemeEval.

Reply:
However, if have RTL ignored, then a viable alternative is to overload this method and have one without RTL parameter.

Reply:
...and to clarify further. I was specifically asked Aryan to implement a flag, so any widget could be marked as `noRTL`

Reply:
After further discussions, here is the conclusion. @aryanrawlani28 could you please add here the following:

```
bool ThemeEval::getWidgetData(const Common::String &widget, int16 &x, int16 &y, int16 &w, int16 &h) {
   bool useRTL;

   return getWidgetData(widget, x, y, w, h, useRTL);
}
```

As a result, we will remove the necessity to modify many places which call this method. I will mark those.